core/src/test/java/org/apache/cxf/bus/ManagedBusTest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.bus;
+
+import javax.management.ObjectName;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.BusFactory;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ManagedBusTest {
```

Comment:
@jgoodyear the `ManagedBus` is covered by `ManagedBusTest` (`systests/uncategorized`)

Reply:
Feels odd to have classes tested outside of the package, even to have some basic smoke testing.

Reply:
CXF is quite old and some decisions as well, but we should not add tests for a sake of tests or/and coverage, it adds significant maintenance costs for no benefits (surely, in context of CXF).

Reply:
Right on.

Shall CXF-9023 & CXF-9024 be closed as 'will not fix'?

Reply:
If you don't mind, let's spend some time and find the gaps, fe. I have not find any tests for `ClientLifeCycleManagerImpl` but you added one (thank again for it), there could be others

Reply:
Take any portion of the PR you see value in retaining :)